### PR TITLE
Bug 1810553 - Update Focus expiring telemetry

### PR DESCRIPTION
**Pull request by t-p-white**
New values provided by @rtestard in Focus 112_expiry_list doc.
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1810553

### CODE REVIEW

File: focus-android/app/metrics.yaml [hunk 1]
```diff
@@ -401,12 +401,13 @@ mozilla_products:
     data_reviews:
       - https://github.com/mozilla-mobile/focus-android/pull/5303
       - https://github.com/mozilla-mobile/focus-android/pull/6315
+      - https://github.com/mozilla-mobile/firefox-android/pull/632
     data_sensitivity:
       - technical
       - interaction
     notification_emails:
       - android-probes@mozilla.com
```
Comment (rocketsroger) [inline focus-android/app/metrics.yaml:409]: Add the product manager's email here for permanent collection.  Thanks

**Comment by t-p-white** [comment]
# Request for Data Collection Renewal
### Renew for 6 months
Total: 2
———
`mozilla_products.is_fenix_default_browser`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/5303

2) When will this collection now expire?
    - never

3) Why was the initial period of collection insufficient?
    - This is a core health indicator we monitor the progress of over time

———
`nimbus_experiments.nimbus_initial_fetch`:
1) Provide a link to the initial Data Collection Review Request for this collection.
    - https://github.com/mozilla-mobile/focus-android/pull/6972

2) When will this collection now expire?
    - 118

3) Why was the initial period of collection insufficient?
    - We need this to validate that experiments inclusive of onboarding can fetch the recipe from Nimbus

———

**Comment by t-p-white** [comment]
# Request for data collection review form
All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.
1. What questions will you answer with this data?
  * This data will help us know how many users have Focus set as the default browser on the user's device.
2. Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements? 
  * This is a core product health indicator that we monitor progress of over time.
3. What alternative methods did you consider to answer these questions? Why were they not sufficient?
  * There are no other alternatives. 
4. Can current instrumentation answer these questions?
  * No. 
5. List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox data collection categories found on the Mozilla wiki. 
Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.

Measurement Description | Data Collection Category | Tracking Bug #
--- | --- | ---
is_fenix_default_browser | Category 1 - Technical data / Category 2 - Interaction data | https://github.com/mozilla-mobile/focus-android/pull/5303

6. Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
  * https://dictionary.telemetry.mozilla.org/apps/focus_android
7. How long will this data be collected? Choose one of the following:
  * No expiry set.
8. What populations will you measure?
  * All channels, all locales, all countries 
9. If this data collection is default on, what is the opt-out mechanism for users?
  * Default Glean SDK opt-out mechanism. 
10. Please provide a general description of how you will analyze this data.
  * Glean and Glam. 
11. Where do you intend to share the results of your analysis?
  * Only on Glean, Glam, and with mobile teams. 
12. Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
   * No third-party tools.

**Comment by rocketsroger** [comment]
> # Request for data collection review form
> 
> All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.
> 
>     1. What questions will you answer with this data?
> 
> 
>     * This will answer the question if Focus users also have Fenix installed and as a default browser.
> 
> 
>     2. Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements?
> 
> 
>     * This will provide us with info regarding the user base for Focus.
> 
> 
>     3. What alternative methods did you consider to answer these questions? Why were they not sufficient?
> 
> 
>     * There are no other alternatives.
> 
> 
>     4. Can current instrumentation answer these questions?
> 
> 
>     * No.    We need a new specific event for a specific app feature.
> 
> 
>     5. List all proposed measurements and indicate the category of data collection for each measurement, using the [Firefox data collection categories](https://wiki.mozilla.org/Data_Collection) found on the Mozilla wiki.    
>        Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.
> 
> 
> Measurement Description 	Data Collection Category 	Tracking Bug #
> Is Fenix the default browser 	Category 2 - Interaction data 	[mozilla-mobile/focus-android#5295](https://github.com/mozilla-mobile/focus-android/issues/5295)
> 
>     6. Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
> 
> 
>     * https://dictionary.telemetry.mozilla.org/apps/focus_android
> 
> 
>     7. How long will this data be collected? Choose one of the following:
> 
> 
>     * No expiry set.
> 
> 
>     8. What populations will you measure?
> 
> 
>     * All channels, all locales, all countries
> 
> 
>     9. If this data collection is default on, what is the opt-out mechanism for users?
> 
> 
>     * Default Glean SDK opt-out mechanism.
> 
> 
>     10. Please provide a general description of how you will analyze this data.
> 
> 
>     * Glean and Glam.
> 
> 
>     11. Where do you intend to share the results of your analysis?
> 
> 
>     * Only on Glean, Glam, and with mobile teams.
> 
> 
>     12. Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
> 
> 
>     * No third-party tools.

# Data Review

1) Is there or will there be **documentation** that describes the schema for the ultimate data set in a public, complete, and accurate way? 

Yes, through the metrics.yaml file and the [Glean Dictionary](https://dictionary.telemetry.mozilla.org/apps/fenix )

2) Is there a control mechanism that allows the user to turn the data collection on and off? 

Yes, through the "Send Usage Data" preference in the application settings

3) If the request is for permanent data collection, is there someone who will monitor the data over time?

Permanent, rtestard@mozilla.com will be monitoring the data

4) Using the **[category system of data types](https://wiki.mozilla.org/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?

Category 2, Interaction data

5) Is the data collection request for default-on or default-off?

default-on

6) Does the instrumentation include the addition of **any *new* identifiers**?

No

7) Is the data collection covered by the existing Firefox privacy notice? 

Yes

8) Does the data collection use a third-party collection tool? 

No

## Result

data-review+

**Comment by rocketsroger** [comment]
> # Request for Data Collection Renewal
> ### Renew for 6 months
> 
> Total: 1 ——— `nimbus_experiments.nimbus_initial_fetch`:
> 
>     1. Provide a link to the initial Data Collection Review Request for this collection.
>        
>        * [Record telemetry for nimbus experiments fetching focus-android#6972](https://github.com/mozilla-mobile/focus-android/pull/6972)
> 
>     2. When will this collection now expire?
>        
>        * 118
> 
>     3. Why was the initial period of collection insufficient?
>        
>        * We need this to validate that experiments inclusive of onboarding can fetch the recipe from Nimbus
> 
> 
> ———

# Data Collection Renewal Review

1) Is the provided Data Collection Review complete, correct, and data-review+ by a Data Steward?

Yes

2) Is the data collection covered by the existing Firefox Privacy Notice?

Yes

## Result

data-review+

**Review by rocketsroger — APPROVED** [review]
🚢

**Pull request merged**